validation/resources.go

```diff
@@ -4,9 +4,16 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
+
 	"github.com/ovrclk/akash/types"
 )
 
+var (
+	ErrNoGroupsPresent = errors.New("validation: no groups present")
+	ErrGroupEmptyName = errors.New("validation: group has empty name")
```

Comment:
let's figure out why this isn't being caught by a linter @troian

Reply:
Fixed. the best option would be to run linter when making commit...